index.html

```diff
@@ -19,25 +19,208 @@
+            <article class="top-part-about-section">
+                <h2 class="about-myself-header">About <br> Myself</h2>
+                <p class="about-myself-paragraph">Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience! If you like what you see and have a
+                    project you need coded, don’t hestiate to contact me.</p>
+                <h2 class="connect-info">LET’S CONNECT</h2>
+                <ul class="social-icons">
+                    <li><a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg"
+                                alt="Github logo"></a></li>
+                    <li><a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg"
+                                alt="Linkedin Icon"></a></li>
+                    <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg"
+                                alt="Angelist logo"></a></li>
+                    <li><a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg"
+                                alt="Twitter_logo"></a></li>
+                    <li><a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg"
+                                alt="Medium logo"></a></li>
+                </ul>
+                <button class="button-resume" type="button">
+                    Get my resume
+                </button>
+            </article>
 
+            <article>
+                <ul class="first-list-about-myself">
+                    <li>Languages</li>
+                    <li> <img src="./assets/icons/Icon-Chevron-Down-Larger.png" alt="Arrow"></li>
+                </ul>
+                <ul class="language-categories-about-myself">
+                    <li class="lang-logo-tags">
+                         <img src="./assets/icons/js-logo-counter.png" alt="Javascript logo" >
+                        <span>Javascript</span>
+                    </li>
+                    <li class="lang-logo-tags">
+                         <img src="./assets/icons/html-logo-counter.png" alt="Javascript logo" >
+                        <span>HTML</span>
+                    </li>
+                    <li class="lang-logo-tags">
+                         <img src="./assets/icons/css-logo-counter.png" alt="Javascript logo" >
+                        <span>CSS</span>
+                    </li>
+                </ul>
+                <ul class="last-list-about-myself">
+                    <li>Frameworks</li>
+                    <li> <img src="./assets/icons/Icon-Chevron-Right-Larger.png" alt="Arrow"></li>
+                </ul>
+                <ul class="last-list-about-myself">
+                    <li>Skills</li>
+                    <li> <img src="./assets/icons/Icon-Chevron-Right-Larger.png" alt="Arrow"></li>
+                </ul>
+            </article>
```

Comment:
Dear Chaw,
This issue that you specified is to add each of them in an li. The first thing that I am concerned about is if I add the languages ul which consists of JS HTML and CSS inside the languages UL won't it disregard their CSS configuration. Because here two parts of my categories need flex-direction: column and flex-direction: row at the same time. Please kindly take that into account and specify it a bit more. Thanks